src/formatDate.js

```diff
@@ -51,6 +51,35 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const newDate = [];
+  const oldSeparator = fromFormat[3];
+  const newSeparator = toFormat[3];
+
+  const arrDate = date.split(oldSeparator);
+
+  const objDate = {};
+
+  for (let i = 0; i <= 2; i++) {
+    objDate[fromFormat[i]] = arrDate[i];
+  }
+
+  if (objDate['YY'] < 30) {
+    objDate['YYYY'] = 20 + objDate['YY'];
+  }
+
+  if (objDate['YY'] >= 30) {
+    objDate['YYYY'] = 19 + objDate['YY'];
+  }
```

Comment:
```suggestion
  if (objDate['YY']) {
    objDate['YYYY'] = (objDate['YY'] < 30 ? 20 : 19) + objDate['YY'];
  }
```

This part can be simplified under one common condition.